bitnami/grafana-alloy/values.yaml

```diff
@@ -0,0 +1,931 @@
+    ##
+    omitEmptySeLinuxOptions: false
+
+## @section Common parameters
+##
+
+## @param kubeVersion Override Kubernetes version
+##
+kubeVersion: ""
+## @param apiVersions Override Kubernetes API versions reported by .Capabilities
+##
+apiVersions: []
+## @param nameOverride String to partially override common.names.name
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param namespaceOverride String to fully override common.names.namespace
+##
+namespaceOverride: ""
+## @param commonLabels Labels to add to all deployed objects
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+## Diagnostic mode
+## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+## @param diagnosticMode.command Command to override all containers in the chart release
+## @param diagnosticMode.args Args to override all containers in the chart release
+##
+diagnosticMode:
+  enabled: false
+  command:
+    - sleep
+  args:
+    - infinity
+
+
+## Configure Pods Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+## @param podSecurityContext.enabled Enable Grafana Alloy pods' Security Context
+## @param podSecurityContext.fsGroupChangePolicy Set filesystem group change policy for Grafana Alloy pods
+## @param podSecurityContext.sysctls Set kernel settings using the sysctl interface for Grafana Alloy pods
+## @param podSecurityContext.supplementalGroups Set filesystem extra groups for Grafana Alloy pods
+## @param podSecurityContext.fsGroup Set fsGroup in Grafana Alloy pods' Security Context
+##
+podSecurityContext:
+  enabled: true
+  fsGroupChangePolicy: Always
+  sysctls: []
+  supplementalGroups: []
+  fsGroup: 1001
+## @param hostAliases Grafana Alloy pods host aliases
+## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+##
+hostAliases: []
+## @param daemonsetAnnotations Annotations for Grafana Alloy daemonset
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+daemonsetAnnotations: {}
+## @param deploymentAnnotations Annotations for Grafana Alloy deployment
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+deploymentAnnotations: {}
+## @param statefulsetAnnotations Annotations for Grafana Alloy statefulset
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+statefulsetAnnotations: {}
```

Comment:
I wonder if it makes sense to have a generic `annotations` or `controllerAnnotations` parameter so we don't need distinguish depending on the resource type.